target/linux/ath79/dts/ar7161_meraki_mr16.dts

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	compatible = "meraki,mr16", "qca,ar7161";
+	model = "Meraki MR16";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
```

Comment:
This can be dropped, it's already in ar7100.dtsi.

Reply:
Removed.